enterpriseknowledgegraph/entity_reconciliation/delete_entity_reconciliation_job_sample_test.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from google.api_core.exceptions import InvalidArgument, NotFound
+
+import delete_entity_reconciliation_job_sample
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "global"
+job_id = "5285051433452986164"
+
+
+def test_delete_entity_reconciliation_job(capsys):
+    try:
```

Comment:
Same as above: remove `try/except`.

Reply:
I added this for the same reason as the others, the job can only be deleted once before it will fail. Is there a recommended practice for this?

Reply:
Same as previous--let's do creation & cancellation in setup, delete the resource in the test, and then set this test to run only nightly.

Side note: do we anticipate that customers will only ever have 2 jobs running concurrently?